src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java

```diff
@@ -58,13 +58,13 @@
 
 public class ItemToolWrench extends ItemBaseTool implements IProvideRecipe {
 
-    public static Set blocksEffectiveAgainst = Sets.newHashSet(new Block[]{});
-    public static boolean canHarvest = false;
-    public static ToolMaterial toolMaterial = EnumHelper.addToolMaterial("GENERITECH_WRENCH", 3, 160, 8.0F, 3.0F, 18);
-    private static IBlockState blockHarvest = null;
+    protected static Set blocksEffectiveAgainst = Sets.newHashSet(new Block[]{});
+    //public static boolean canHarvest = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)